lib/v2/bilibili/liveSearch.js

```diff
@@ -15,26 +17,33 @@ module.exports = async (ctx) => {
             orderTitle = '人气直播';
             break;
     }
+    const verifyString = await cache.getVerifyString(ctx);
+    let params = `__refresh__=true&_extra=&context=&page=1&page_size=42&order=online&duration=&from_source=&from_spmid=333.337&platform=pc&highlight=1&single_column=0&keyword=${urlEncodedKey}&qv_id=68nr2HBn3S27waWVjvFT0CLT7U3TF0U3&ad_resource=&source_tag=3&gaia_vtoken=&category_id=&search_type=live&dynamic_offset=0&web_location=1430654`;
+    params = utils.addVerifyInfo(params, verifyString);
 
     const response = await got({
         method: 'get',
-        url: `https://api.bilibili.com/x/web-interface/search/type?search_type=live_room&keyword=${urlEncodedKey}&order=${order}&coverType=user_cover&page=1`,
+        url: `https://api.bilibili.com/x/web-interface/wbi/search/type?${params}`,
         headers: {
-            Referer: `https://search.bilibili.com/live?keyword=${urlEncodedKey}&order=${order}&coverType=user_cover&page=1&search_type=live`,
+            Referer: `https://search.bilibili.com/live?keyword=${urlEncodedKey}&from_source=webtop_search&spm_id_from=444.7&search_source=3&search_type=live_room`,
+            'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36',
```

Comment:
RSSHub will use a random UA of Chrome on a macOS desktop.
https://github.com/DIYgod/RSSHub/blob/cb7ea448b937c2164ec0bd8d82924567a7e6596d/lib/config.js#L49
Are there any specific reasons to use a fixed UA of Chrome on Windows 10?